nixos/modules/services/backup/postgresql-backup.nix

```
@@ -87,7 +102,7 @@ in {
         default = "/var/backup/postgresql";
         type = types.path;
         description = ''
-          Location to put the gzipped PostgreSQL database dumps.
+          Location to put the compressed PostgreSQL database dumps.
```

```suggestion
          Directory path to put the PostgreSQL database dumps.
```

They don't have to be compressed anymore.